processing/src/main/java/org/apache/carbondata/processing/datatypes/ArrayDataType.java

```diff
@@ -171,9 +175,9 @@ public void fillCardinality(List<Integer> dimCardWithComplex) {
   /**
    * parse byte array and bit pack
    */
-  @Override
-  public void parseAndBitPack(ByteBuffer byteArrayInput, DataOutputStream dataOutputStream,
-      KeyGenerator[] generator) throws IOException, KeyGenException {
+  @Override public void parseAndBitPack(ByteBuffer byteArrayInput,
```

Comment:
Now this method is not used? if not delete it

Reply:
Removed